pep-8016.rst

```diff
@@ -291,14 +299,19 @@ maintain Python, core team members who have stopped contributing are
 encouraged to declare themselves as "emeritus members". Those who
 haven't made any non-trivial contribution in two years may be asked to
 move themselves to this category, and moved there if they don't
-respond. Emeritus team members lose their privileges such as voting
-rights and commit access. If an emeritus team member later resumes
-contributing, they may rejoin the core team on request, without the
-need for a vote.
+respond. To record and honor their contributions, emeritus team
```

Comment:
You might consider replacing "Emeritus" with "Inactive" :-)
https://discuss.python.org/t/pep-8016-the-steering-council-model/394/28